tools/ichaindis.py

```diff
@@ -19,11 +20,53 @@
     'ICHAIN_VEC3S',
 ]
 
+Z64_ACTOR_PATH = "../include/z64actor.h"
```

Comment:
Correct me if I'm wrong, but doesn't this assume running the script from a certain directory? I think we should make paths relative to the script's directory, not the current user directory. See `m2ctx.py` for an example

Reply:
Yeah it does, I've fixed it now to make it relative to the script's location so if the script ever moves it will need to be updated but it should work even if called from another folder.

Reply:
Thanks! :)